graphics/src/MeshManager.cc

```diff
@@ -146,20 +149,22 @@ const Mesh *MeshManager::Load(const std::string &_filename)
         extension.begin(), ::tolower);
     MeshLoader *loader = nullptr;
 
-    loader = &this->dataPtr->assimpLoader;
-    /*
     if (extension == "stl" || extension == "stlb" || extension == "stla")
-      loader = &this->dataPtr->stlLoader;
+      if (this->dataPtr->useAssimp)
+        loader = &this->dataPtr->assimpLoader;
+      else
+        loader = &this->dataPtr->stlLoader;
     else if (extension == "dae")
```

Comment:
Can we also use it for DAE and OBJ if the user requests so? This should make it easier to compare both loaders.

Reply:
@ahcorde, 
Should I close this PR and the PR related the server flag, and create a new one that uses environment variable? The ability the change the assimp flag during runtime sounds good.